lib/base-compiler.js

```diff
@@ -61,6 +61,9 @@ Compile.prototype.couldHaveOptOutput = function (options, version) {
     return version.toLowerCase().indexOf("clang") > -1 &&
         options.some((x) => x == "-fsave-optimization-record");
 };
+Compile.prototype.couldHaveASTOutput = function (options, version) {
```

Comment:
It doesn't appear this function is used? Could it be that you should use this below (see further note)